base/domain-sot.yml

```diff
@@ -0,0 +1,380 @@
+      claim: release
+      text: |
+        Re ya leboha ha o arabile dipotso tsohle.
+
+        Ho ipapisitswe le dikarabo tsa hao, o kotsing e tlase ya ho ba le COVID-19 mme ho ke ke ha hlokahala hore o phethe tlhahlobo ena hape matsatsing a 7 a tlang.
+        *Haeba* o qala ho kula kapa o kopana le motho ya tshwaeditsweng ke COVID-19 nakong ya matsatsi a 7 a latelang, ka kopo nka tlhahlobo ena ya kotsi hape
+
+        🤒 Araba ka *MATSHWAO* bakeng sa dintlha tse ding
+        📌 Araba ka *MENU* ho kgutlela ho menu wa pele
+
+  utter_risk_moderate:
+    - image: "https://healthcheck-rasa-images.s3.af-south-1.amazonaws.com/receipt-mod.png"
+      claim: release
+      text: |
+        *Ho ipapisitswe le dikarabo tsa hao, re kgothaletsa hore o ikarole le batho hofihlela matsatsi a 14 a tlang a fela mme o tlatse HealthCheck ena letsatsi le letsatsi ho hlahloba matshwao a hao.*
+
+        Haeba ho kgonahala, dula o be o robale o le mong ka kamoreng e nang le fesetere e nang le moya o phallang.
+        - O ka bua le ba bang ba lelapa le ho tswela kantle, empa o lokela ho ema bonyane dimithara tse 2 ho motho e mong le e mong ka dinako tsohle.
+        - Ha wa lokela ho ba le baeti ntlong ya hao nakong ena.
+        - Setho se le seng sa lelapa se lokela ho abelwa ho ba mohlokomedi wa hao. Sena se beha karolo e tlase ya lelapa kotsing mme hape se thusa hore mohlokomedi wa hao a be le mekgwa e metle ya tlhokomelo ho qoba hore a se fumane tshwaetso ya vaerase ho tswa ho wena.
+        - Haeba tsena di sa kgonehe, o lokela ho letsetsa mosebeletsi wa tlhokomelo ya bophelo bo botle kapa tleliniki bakeng sa mekgwa e meng.
+
+        Araba ka * QUARANTINE * bakeng sa dintlha malebana le ho ikarohanya le batho
+        📌Araba ka *HO HLAHLOBA* bakeng sa dintlha tse ding kapa *MENU* ho kgutlela ho menu wa pele
+
+  utter_risk_high:
+    - image: "https://healthcheck-rasa-images.s3.af-south-1.amazonaws.com/receipt-high.png"
+      claim: release
+      text: |
+        *Ho ipapisitswe le dikarabo tsa hao, o ka fumanwa o lokela ho hlahlojwa ho tseba hore na o tshwaeditswe ke COVID-19.*
+
+        Letsetsa Hotline ya COVID-19: 08000299, kapa ngaka ya hao ya bophelo bo botle bakeng sa ditlhakisetso hore na o lokela ho latela mehato efe, jwang le hore na o ka hlahlobela kae haeba o kgothaleditswe ho etsa jwalo.
+
+        Latela mehato ena ya bohlokwa ha o batla tlhokomelo:
+        - Qoba ho kopana le batho ba bang ka tlung le tikolohong eo o phelang ho yona, haeba ho kgonahala, dula kamoreng e arohaneng le tse ding.
+        - Etsa bonnete ba hore o rwala maske wa sefahleho pele o kena setsing sa tlhokomelo ya bophelo bo botle.
+
+        Haeba o na le, kapa o qala ho ba le matshwao a matla, batla tlhokomelo ya bongaka ka potlako.
+
+        Araba ka * QUARANTINE * bakeng sa dintlha malebana le ho ikarohanya le batho
+        📌Araba ka *HO HLAHLOBA* bakeng sa dintlha tse ding kapa *MENU* ho kgutlela ho menu wa pele
```

Comment:
This keyword is just `HLAHLOBA` in the turn automation

Reply:
`MENU` will result in them seeing the english menu

Reply:
@rudigiesler What would you suggest be done in a case we do not have a SOT_ZA translation of the word/ that is the direct translation?

Reply:
Looking at the Turn automations, `KA-MENU` is the keyword for the main menu. So `KA-MENU` will take you to the `sot` menu, `MENU` will take you to the `eng` menu